cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -71,7 +71,9 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
    */
   function init() {
     var triggerClickedBinded = _triggerClicked.bind( this );
+    var triggerOverBinded = _triggerOver.bind( this );
```

Comment:
Oops, realized why I did this, so there's a call to the binded `_triggerClicked` in both the trigger and alternative trigger. The `Binded` suffix is for cases like https://github.com/cfpb/cfgov-refresh/pull/1572/files#diff-982f166b46c3d02b0eb18e05e6f3e66fR60, where the binded function needs to be shared across methods internally but can't have the same name as the function its binding. Doesn't need to be a `Binded` suffix, but it needs to be some difference in the name.